pyproject.toml

```diff
@@ -6,7 +6,7 @@ build-backend = "setuptools.build_meta"
 name = "eventio"
 description = "Python read-only implementation of the EventIO file format"
 
-requires-python = '>=3.10'
+requires-python = '<3.13'
```

Comment:
no, we are compatible with 3.13. And we do require at least 3.10.

This is why the CI is failing to try to build wheels for 3.6

Reply:
reverted